### PR TITLE
add enterprise terraform with kubernetes docs

### PR DESCRIPTION
**Pull request by wasaga**
https://github.com/pomerium/internal/issues/1970

### CODE REVIEW

File: content/docs/enterprise/install.mdx [hunk 1]
```diff
@@ -211,5 +211,34 @@ kubectl apply -k ./config
...
+
+This file contains the essential parameters for Pomerium Enterprise deployment.
+
+<CodeBlock language="terraform" title="locals.tf">
+  {TerraformLocals}
+</CodeBlock>
+
+### The Main Terraform File
+
+<CodeBlock language="terraform" title="locals.tf">
```
Comment (gaurdro) [inline content/docs/enterprise/install.mdx:233]: Probably want to change the filename for this and below.

File: content/docs/enterprise/tf/main.tf [hunk 2]
```diff
@@ -0,0 +1,40 @@
+provider "kubernetes" {}
+
+provider "random" {}
+
+module "pomerium_enterprise" {
+  source = "git::https://github.com/pomerium/documentation//terraform/enterprise-k8s?ref=v0.28.0"
```
Comment (gaurdro) [inline content/docs/enterprise/tf/main.tf:6]: It feels weird to have this module in our docs repo.  Maybe put it in pomerium/pomerium?

Comment (wasaga) [reply content/docs/enterprise/tf/main.tf:6]: well, that's for the enterprise console - might be little weird to have it in pomerium/pomerium either, no?

**Review by gaurdro — APPROVED** [review]
Approving to unblock the user and I don't see anything technically wrong or harmful.

**Comment by wasaga** [comment]
superceded by https://github.com/pomerium/install/pull/5

**Pull request closed without merging**